Makefile-docker

```diff
@@ -1,3 +1,5 @@
+export FXA_EMAIL=uitest-$(shell uuid)@restmail.net
+export FXA_PASSWORD=uitester
```

Comment:
Let's add a comment what this is used for. Maybe even try to put this directly into the `ui-test` action but I'm not sure if make allows this.